### ISSUE TITLE
Excellent work! What is the procedure of GET during inference？

### ISSUE THREAD

**Issue by sfengpeng** [enguangW/GET#3]

**Comment by enguangW** [comment]
Thank you for your kind words! During inference, given an image, its visual or text features are used to obtain clustering assignments through a classifier.  You can see the relevant code [here](https://github.com/enguangW/GET/blob/8e908c061660ce95b07596b9a7f190e5c444f7cb/s2_GET.py#L309). In our approach, we utilize the visual features for the inference process.

**Comment by sfengpeng** [comment]
> Thank you for your kind words! During inference, given an image, its visual or text features are used to obtain clustering assignments through a classifier. You can see the relevant code [here](https://github.com/enguangW/GET/blob/8e908c061660ce95b07596b9a7f190e5c444f7cb/s2_GET.py#L309). In our approach, we utilize the visual features for the inference process.

Thanks for your reply. I read the code and wonder why you don't utilize both visual and text features for inference?  Intuitively, this strategy can further improve performance.

**Comment by enguangW** [comment]
Thanks for your insightful question! In our initial experiments, we focused on using visual features as the primary branch. We did observe that in some datasets, the results from the text branch could be higher. However, since we did not implement distillation at the logits level (as done in approaches like TextGCD), there may be some misalignment between the visual and text outputs. This misalignment could lead to suboptimal performance when combining both modalities on certain datasets. To avoid the complexities of tuning and method design, we ultimately decided to rely solely on the visual results. We appreciate your understanding and your interest in our work!
